crates/forge_permission_resolver/src/permissions_resolver.rs

```diff
@@ -362,4 +390,41 @@ mod test {
             "Should require admin:repository:bitbucket permission"
         );
     }
+
+    #[test]
+    fn test_get_issues_for_epic() {
+        let (permission_map, regex_map) = get_permission_resolver_jira_software();
+        let url = "/rest/agile/1.0/sprint/23";
+        let request_type = RequestType::Get;
+        let result = check_url_for_permissions(&permission_map, &regex_map, request_type, url);
+
+        println!("Permission Map: {:?}", permission_map);
```

Comment:
@jwong101 @jwong33 should we remove debug text ?

Reply:
Yeah, this should be removed.

Reply:
Yeah, I think that'd be good. I see there is some from my code too 😬 😓